scripts/local-to-mainnet/commands.sh

```diff
@@ -31,13 +31,13 @@ docker-compose logs -f relayer-host | sed -r -u "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2}
 
 #### REGISTER HOST
 # IBC Transfer from HOST to stride (from relayer account)
-build/osmosisd tx ibc-transfer transfer transfer $transfer_channel stride1u20df3trc2c2zdhm8qvh2hdjx9ewh00sv6eyy8 4000000uosmo --from hot --chain-id osmosis-1 -y --keyring-backend test --node http://osmo-fleet-direct.main.stridenet.co:26657
+build/osmosisd tx ibc-transfer transfer transfer $transfer_channel $STRIDE_ADMIN_ADDRESS 4000000uosmo --from hot --chain-id osmosis-1 -y --keyring-backend test --node http://osmo-fleet-direct.main.stridenet.co:26657
```

Comment:
these scripts in `local-to-mainnet` are actually not related to dockernet - can you revert these? (same thing in `templates/commands.sh`)